k8s/backend-configs/apache-tika-backend-config.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: cloud.google.com/v1
+kind: BackendConfig
+metadata:
+  name: apache-tika-backendconfig
+spec:
+  timeoutSec: 300
```

Comment:
Do we need a super long timeout ?

Reply:
Happy to have a lower one but FWIW, either it works or it breaks pretty quickly.

Reply:
Yeah no opinion here really, was just wondering since this is cargo culted from other backendconfigs where we had to override timeout for specific reasons (upserting docs in core can take a very long time)

Reply:
Lowered to 120s.